assets/dev/js/editor/document/hooks/ui/create/move-column-resize-handle.js

```diff
@@ -0,0 +1,46 @@
+import After from 'elementor-api/modules/hooks/ui/after';
+
+export class MoveColumnResizeHandle extends After {
```

Comment:
It shouldn't be applied by a hook. It should happen where you create the resize functionality

Reply:
It's OK because it's a change from the normal flow.